api/config/v2alpha2/projectconfig_types.go

```diff
@@ -38,9 +38,12 @@ type ProjectConfig struct {
 	// protection if it is not set on the resource.
 	DeletionProtectionDefault bool `json:"deletionProtectionDefault"`
 
-	// ReadOnly sets the default values of ReadOnly for systems
+	// ReadOnly sets the value of ReadOnly for systems
 	ReadOnly bool `json:"readOnly"`
 
+	// EnableDeltaBundlesDefault sets the default of whether systems have delta-bundles or not
+	EnableDeltaBundlesDefault *bool `json:"enableDeltaBundlesDefault,omitempty"`
```

Comment:
Is there a reason why it is a pointer to a bool and not just a bool?

Reply:
Ellers kan man ikke distinguish false fra unset